libs/ui/src/link_button.tsx

```diff
@@ -1,38 +1,52 @@
 import { useHistory } from 'react-router-dom';
-import { Button, ButtonProps } from './button';
+import { forwardRef, Ref, SyntheticEvent } from 'react';
+import { Button, ButtonProps, ClickHandler, TypedClickHandler } from './button';
 
 /**
  * Props for {@link LinkButton}.
  */
 export interface LinkButtonProps extends Omit<ButtonProps, 'onPress'> {
+  // export type LinkButtonProps = ButtonProps & {
   goBack?: boolean;
   primary?: boolean;
   to?: string;
+  ref?: Ref<Button<SyntheticEvent>>;
```

Comment:
It doesn't seem like `SyntheticEvent` should be passed in as the value type of the `Button` - isn't that type parameter only used when storing a `value` on the button?

Reply:
Summarizing chat: `SyntheticEvent` value type was passed here to avoid `Type 'Ref<Button<undefined>>' is not assignable to type 'LegacyRef<Button<SyntheticEvent<Element, Event>>> | undefined'.` error in `LinkButton`. Not sure exactly how this type inference was working, but passing `never` fixes the problem. I believe this is correct because `LinkButton` never passes an `onClick` that requires a default value.

https://github.com/votingworks/vxsuite/pull/4284/commits/2046a88da5ba818c4b6ecf87d7149b0f11ae2710#diff-4ed014966e4ba63b7cc24fa7d03553eb0d1ded7d73a9ef1641d6c8df5e6a8c5eR44